.vscode/launch.json

```diff
@@ -1,45 +1,33 @@
-  {
+{
     "version": "0.1.0",
     "configurations": [
         {
             "name": "Debug - Openocd docker Remote",
-            "type":"cortex-debug",
-            "cortex-debug.armToolchainPath":"${env:ARM_NONE_EABI_TOOLCHAIN_PATH}/bin",
+            "type": "cortex-debug",
             "cwd": "${workspaceRoot}",
             "executable": "${command:cmake.launchTargetPath}",
             "request": "launch",
             "servertype": "external",
-            // This may need to be arm-none-eabi-gdb depending on your system
-            "gdbPath" : "${env:ARM_NONE_EABI_TOOLCHAIN_PATH}/bin/arm-none-eabi-gdb",
+            // FIXME: This is hardcoded. I have no idea how to use the values set in build.sh here
```

Comment:
Those hard coded paths replace currently working setups that use the environment variables. This is a blocking issue.

Maybe instead of overwriting currently available configs add a new one for the dev container? If this is possible

Reply:
I will add another launch configuration for launching in a devcontainer. That does not solve the problem of having the path/version be hardcoded but it will not break the existing workflows, which rely on the environment variables.